### ISSUE TITLE
Pretraining script for LayoutLM

### ISSUE THREAD

**Issue by viantirreau** [microsoft/unilm#263]
Hi!

First of all, thanks for this excellent work! I had great results replicating the FUNSD fine-tuning example.
I have a large unlabeled dataset of words with their corresponding bounding boxes. I think I could leverage a great performance increase by pretraining on said dataset and then fine-tuning with a few manually labeled examples for sequence labeling tasks. 

I was wondering if you could provide us a rough pretraining script for the Masked Visual-language Model task, for example the one used for your paper. If it is not possible, could I ask you for some orientation on how to approach this?

Thanks in advance!

**Comment by wolfshow** [comment]
@viantirreau We do not have plan to release the pre-training script. I suggest you may read the BERT implementation for pre-training and modify our fine-tuning code.

**Comment by viantirreau** [comment]
Ok, thanks!!
